src/run.py

```diff
@@ -170,9 +170,9 @@ def review_pr(self, pr: GitPullRequest, pr_url: str):
 						# Handle edit.
 						for block in file_diff.diff['blocks']:
 							change_type = block['changeType']
-							if change_type == 0 or change_type == 2 or change_type == 3:
+							if change_type == 0 or change_type == 2:
 								continue
-							assert change_type == 1, f"Unexpected change type: {change_type}"
+							assert change_type == 1 or change_type ==3, f"Unexpected change type: {change_type}"
```

Comment:
If the PR branch is not up to date with the target branch, then it seems like this change makes the script think that lines in the PR branch that were already corrected in the target branch are new/changed lines and introduce a change. This can make the script comment on unchanged lines.